lighthouse-core/report/styles/report.css

```diff
@@ -307,15 +320,19 @@ body {
 }
 
 .report-body__header {
-  height: 58px;
+  height: var(--report-header-height);
   border-bottom: 1px solid #EBEBEB;
   background: #FAFAFA;
-  margin-left: 22%;
+  margin-left: var(--report-menu-width);
   display: flex;
   flex-direction: row;
   align-items: center;
   justify-content: flex-end;
   padding: 0 var(--heading-line-height);
+  position: fixed;
+  z-index: 1;
```

Comment:
do me a solid and add `will-change: transform` on here as well? for lo-dpi chrome desktop users.